### PR TITLE
Update source setup - Closes #642

### LINKED ISSUE #642: Update setup from source
## Which content is outdated?
- (Source code install) Postgres is also needed
- (Source code install) Freegeoip is outdated

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #642

### CODE REVIEW

File: modules/ROOT/pages/setup/source.adoc [hunk 1]
```diff
@@ -67,6 +71,104 @@ brew install node@12.15.0
...
+
+WARNING: If other versions of PostgreSQL are currently installed on your machine, please ensure they are stopped before starting the docker container.
+
+[source,bash]
+----
+docker run --name lisk_service_db -p 5432:5432 -e POSTGRES_USER=lisk -e POSTGRES_PASSWORD=password -e POSTGRES_DB=lisk -d postgres:10
+----
+
+This will install PostgreSQL version 10 (`postgres:10`) in a container with name `lisk_service_db` and binds the port `5432` of the container with the same port of the machine.
+As environment variables `POSTGRES_USER=lisk` is exposed in order to create the Lisk user, and `POSTGRES_PASSWORD=password` is used to set the password for the lisk user.
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
As environment variables `POSTGRES_USER=lisk` is exposed in order to create the Lisk user, and `POSTGRES_PASSWORD=password` is used to set the password for the Lisk user.
```

File: modules/ROOT/pages/setup/source.adoc [hunk 2]
```diff
@@ -67,6 +71,104 @@ brew install node@12.15.0
...
+[tabs]
+====
+Postgres with Docker::
++
+--
+Running Postgres inside a Docker container will setup the correct version of Postgres and containerize it away from any existing versions that may exist locally on your machine.
+Choose this setup if you are unfamiliar with Postgres, or if other issues occur with a previously installed version of Postgres.
+To successfully perform the command shown below, install Docker as described in the Setup page of the xref:{url_docker_setup][Lisk Core Docker image].
+
+WARNING: If other versions of PostgreSQL are currently installed on your machine, please ensure they are stopped before starting the docker container.
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
WARNING: If other versions of PostgreSQL are currently installed on your machine, please ensure they are stopped before starting the Docker container.
```

File: modules/ROOT/pages/setup/source.adoc [hunk 3]
```diff
@@ -67,6 +71,104 @@ brew install node@12.15.0
...
+docker restart lisk_service_db <3>
+docker rm lisk_service_db <4>
+----
+
+<1> Stop the container.
+<2> Start the container.
+<3> Restart the container.
+<4> Remove the container.
+
+In order to access Postgres inside the container via CLI, run the following command displayed below:
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
In order to access Postgres inside the container via the CLI, execute the following command displayed below:
```

File: modules/ROOT/pages/setup/source.adoc [hunk 4]
```diff
@@ -67,6 +71,104 @@ brew install node@12.15.0
...
+
+[source,bash]
+----
+sudo pg_dropcluster --stop 10 main
+sudo pg_createcluster --locale en_US.UTF-8 --start 10 main
+----
+
+Create a new database user called `lisk` and grant it rights to create databases.
+Then create the database with the `lisk` user as owner.
+In the last step, define the password for the `lisk` user:
```
Comment (chris529) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
Finally, define the password for the `lisk` user:
```

**Review by chris529 — COMMENTED** [review]
Looks very good, just a few minor points 👍

File: modules/ROOT/pages/setup/source.adoc (hunk 1 above)
Comment (Tschakki) [inline modules/ROOT/pages/setup/source.adoc]:
```suggestion
As environment variables `POSTGRES_USER=lisk` is exposed in order to create the Lisk user, and `POSTGRES_PASSWORD=password` is used to set the password for the `lisk` user.
```

**Review by chris529 — APPROVED** [review]
Looks great! 👍

**Pull request merged**